binance-dex/adapter.js

```diff
@@ -0,0 +1,58 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const apiEndpoint = process.env.API_ENDPOINT || 'dex-asiapacific'
```

Comment:
It would be more readable if the default options were specified at the start:

```suggestion
const DEFAULT_API_ENDPOINT = 'dex-asiapacific'
const DEFAULT_DATA_ENDPOINT = 'v1/ticker/24hr'
const apiEndpoint = process.env.API_ENDPOINT || DEFAULT_API_ENDPOINT
```